packages/maskbook/src/plugins/BarnBridge/UI/YieldFarmingView.tsx

```diff
@@ -0,0 +1,235 @@
+                            <BondTokenIconCircle className={classes.tapBarCellRowIcon} />
+                        </Paper>
+                    </Paper>
+                </Paper>
+
+                <Grid className={classes.gridRoot} container spacing={1}>
+                    <Grid item xs={6}>
+                        <Paper className={classes.gridPaper}>
+                            <Typography className={classes.gridCellTitle} variant="caption">
+                                {t('plugin_barnbridge_yield_farming_tvl')}
+                            </Typography>
+                            <Typography className={classes.gridCellValue} variant="h5">
+                                $12,345,678
+                            </Typography>
+                            <Typography className={classes.tapBarCellTitle} variant="caption">
+                                $101,192,261 effective locked
+                            </Typography>
+                        </Paper>
+                    </Grid>
+                    <Grid item xs={6}>
+                        <Paper className={classes.gridPaper}>
+                            <Typography className={classes.gridCellTitle} variant="caption">
+                                {t('plugin_barnbridge_yield_farming_bond_rewards')}
+                            </Typography>
+                            <Typography className={classes.gridCellValue} variant="h5">
+                                10,000,000
+                            </Typography>
+                            <Typography className={classes.tapBarCellTitle} variant="caption">
+                                out of 2,860,000
+                            </Typography>
+                        </Paper>
+                    </Grid>
+                    <Grid item xs={6}>
+                        <Paper className={classes.gridPaper}>
+                            <Typography className={classes.gridCellTitle} variant="caption">
+                                {t('plugin_barnbridge_yield_farming_bond_price')}
+                            </Typography>
+                            <Typography className={classes.gridCellValue} variant="h5">
+                                $24.08
+                            </Typography>
+                            <Typography className={classes.tapBarCellTitle} variant="caption">
+                                Uniswap market
+                            </Typography>
+                        </Paper>
+                    </Grid>
+                    <Grid item xs={6}>
+                        <Paper className={classes.gridPaper}>
+                            <Typography className={classes.gridCellTitle} variant="caption">
+                                {t('plugin_barnbridge_yield_farming_time_left')}
+                            </Typography>
+                            <Typography className={classes.gridCellValue} variant="h5">
+                                2d 18h 42m 12s
```

Comment:
why hard-coded?

Reply:
This tab is disabled. I wrote the UI for it but the data source is not hooked up. That is another feature which will happen later.

Reply:
So it's all hardcoded until then? If you prefer I parameterize it before landing i can do that.